Makefile

```diff
@@ -31,6 +31,10 @@ clean:
 distclean: clean
 	find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
 	rm -f manifest/*.yaml
+	rm -f .glide.status
+
+checksync: .glide.status
+	test -d vendor || make sync
 
 sync:
 	glide install --strip-vendor
```

Comment:
Sorry for asking so many questions, but is not `glide install` the one which takes only glide.lock into account (with other words the cheaper operation?)? AFAIK `glide update` is more expensive, since  it goes through glide.yaml, checks out all dependencies, goes through all subdependencies there, prepares the vendor folder, and finally updates the `glide.lock` file.